### PR TITLE
Add E-Ink display support

### PR DESCRIPTION
**Pull request by Coscolin**

### CODE REVIEW

File: CO2_Gadget_EINK.h [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+#include "icons.h"
+#define EPD_SCLK SCK  // 18
+#define EPD_MISO 17
+#define EPD_DC 17      // MISO
+#define EPD_MOSI MOSI  // 23
+#define EPD_CS SS
+#define EPD_RST 16
+#define EPD_BUSY 4
+
+#include "bitmaps/Bitmaps104x212.h"  // 2.13" b/w flexible GDEW0213I5F
```
Comment (melkati) [inline CO2_Gadget_EINK.h:32]: What is the content?
These files are not included in PR (and I don't have any bitmaps folder)

File: CO2_Gadget_EINK.h
Comment (melkati) [inline CO2_Gadget_EINK.h:1]: There are missing fonts on the PR

File: CO2_Gadget_EINK.h (hunk 1 above)
Comment (melkati) [reply CO2_Gadget_EINK.h:32]: I investigated this a little and looks as a leftover from the GxEPD2 library examples (it's loading these files from folders in the libraries)

Comment (Coscolin) [reply CO2_Gadget_EINK.h:32]: I think you can delete line 32.

File: CO2_Gadget_EINK.h
Comment (Coscolin) [reply CO2_Gadget_EINK.h:1]: NotoSans_Bold42pt7b.h and NotoSans_Bold48pt7b.h came from your branch low-power-eink

**Pull request merged**